content/blog/2023/03/camel-k-telemetry/index.md

```diff
@@ -0,0 +1,420 @@
+---
+title: "Camel K Observability: Distributed Tracing"
+date: 2023-03-25
```

Comment:
Any reason why to post this in 2 weeks!?

Reply:
None, I was just giving myself enough time. But that's also something I wanted to discuss : the date and when to publish/communicate on it.

Reply:
IMO the blog is already good to go. We don't have any other blog in the pipeline and the last one was published a couple of weeks ago, so we can publish today itself.